src/Models/IoTWorkspaceProject.ts

```diff
@@ -461,10 +463,36 @@ export class IoTWorkspaceProject extends IoTWorkbenchProjectBase {
       }
     }
 
+    // Configure project and open in container
+    const projectEnvConfiger = new ProjectEnvironmentConfiger();
+    projectEnvConfiger.configureProjectEnvironmentCore(
+        this.extensionContext, this.channel, this.telemetryContext, deviceDir,
+        PlatformType.Arduino, openInNewWindow);
+  }
+
+  async configureProjectEnv(
+      channel: vscode.OutputChannel, scaffoldType: ScaffoldType,
+      configureRootPath: string, templateFilesInfo: TemplateFileInfo[],
+      openInNewWindow: boolean, customizeEnvironment: boolean) {

```

Comment:
>customizeEnvironment: boolean [](start = 32, length = 29)

What is the "customizeEnvironment" parameter for?

Reply:
Will fix.